From 329801ffe7fe563bd123a87bd44a9328cc4751f6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fabiano=20Fid=C3=AAncio?= <fidencio@redhat.com>
Date: Thu, 30 Jun 2016 11:18:01 +0200
Subject: [PATCH] remote-viewer: Use a different mnemonic for Connection
 Address widget

Alt+C clashes with the mnemonic used for the Cancel button.

Related: rhbz#1351487
---
 src/remote-viewer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/remote-viewer.c b/src/remote-viewer.c
index a826594..78d2d25 100644
--- a/src/remote-viewer.c
+++ b/src/remote-viewer.c
@@ -1084,7 +1084,7 @@ connect_dialog(GtkWindow *main_window, gchar **uri)
     gtk_container_set_border_width(GTK_CONTAINER(box), 5);
     gtk_box_pack_start(GTK_BOX(area), box, TRUE, TRUE, 0);
 
-    label = gtk_label_new_with_mnemonic(_("_Connection Address"));
+    label = gtk_label_new_with_mnemonic(_("Connection _Address"));
     gtk_misc_set_alignment(GTK_MISC(label), 0, 0.5);
     gtk_box_pack_start(GTK_BOX(box), label, TRUE, TRUE, 0);
     entry = GTK_WIDGET(gtk_entry_new());
